app/controllers/comments_controller.rb

```diff
@@ -1,17 +1,31 @@
 class CommentsController < ApplicationController
   def new
-    @comment = Comment.new
-    @post = Post.find(params[:post_id])
+    @posts = Post.find(params[:post_id])
+    @user = User.find(params[:user_id])
+    @comments = Comment.find(params[:comment_id])
+
+    respond_to do |format|
+      format.html
+      format.json { render json: @comments }
+    end
```

Comment:
- [ ] Nice adding JSON format but when I try to access the posts or comments from API app like Thunder or Postman I got the following error because csrf token prevent requests from an unknown source 
I suggest adding URLs for API separated from normal front-end app URLs 
<img width="1410" alt="Screenshot 2023-05-25 at 23 10 35" src="https://github.com/hakichris/Blog-App/assets/61790587/630fe89f-bd17-427f-b8a2-a710fb6d4aed">

Reply:
thank you @youmari, working on it.

Reply:
I fixed it. @youmari